site_tmpl/uglydetail.html

```diff
@@ -24,6 +24,11 @@ <h3>Status: <b>{{ event.status }}</b></h3>
                 {% if request.user.is_lnl %}
                     <a class="btn btn-info btn-lg" target="_blank" href="https://wpi0.sharepoint.com/sites/gr-lnl/Event%20Photos/Forms/Thumbnails.aspx?view=7&q={{ event.event_name|urlencode }}">Photos</a>
                 {% endif %}
+                {% if event.event_id %}
+                        <tr>
+                            <a class="btn btn-primary btn-lg" target="_blank" href="https://25live.collegenet.com/pro/wpi#!/home/event/{{ event.event_id }}/details">25Live</a>
+                        </tr>
+                        {% endif %}
```

Comment:
I'm not sure I like how much clutter this adds to the top bar, but I understand that there aren't many alternatives 
![screenshot-2022-09-12-095846](https://user-images.githubusercontent.com/8702646/189673542-35944d5d-857a-436c-9e13-5b31ed708791.png)

Reply:
Agreed. We could consider moving the `PDF` button to a different location given it's very infrequently used.

Reply:
I would also be open to moving `Photos`, since that's more kludgy (it only opens the Sharepoint search)